core/presets/configs.go

```diff
@@ -112,9 +112,9 @@ questions:
       message: What javascript package manager do you want to use
       options:
         - name: npm
-          template: npm-laravel.yml
```

Comment:
@fabriciojs - the reason we have `-laravel` versions of the `npm` and `yarn` scripts is because the `laravel.yml` template is the only one that requires a `node-setup` script (which is different for `npm` vs. `yarn`). Based on these changes, the `setup` and `reset` scripts in `kool.yml` are now broken.

https://github.com/kool-dev/kool/blob/4e159861d4b8ff1f14910787e8c49ca27750531b/templates/scripts/laravel.yml#L11

https://github.com/kool-dev/kool/blob/4e159861d4b8ff1f14910787e8c49ca27750531b/templates/scripts/laravel.yml#L16